src/Controllers/HelloWorldController.cs

```diff
@@ -1,17 +1,29 @@
 using Microsoft.AspNetCore.Mvc;
 using System.Text.Encodings.Web;
+using Microsoft.AspNetCore.Mvc.Localization;
 
 namespace TrainingNet.Controllers
 {
     [Route("[controller]")]
     public class HelloWorldController : Controller
     {
+        private readonly IHtmlLocalizer<HelloWorldController> _localizer;
+        
+        public IHtmlLocalizer<HelloWorldController> Localizer
+            {
+                get {return this._localizer;}
```

Comment:
dont forget spacing between brakcets.
also, this could be written as a body expression
```csharp
get => this._localizer;
```